routes/web.php

```diff
@@ -164,9 +164,8 @@
 
 Route::get('/manageSubProject.php', 'SubProjectController@manageSubProject');
 
-Route::get('/viewSubProjectDependenciesGraph.php', 'SubProjectController@dependenciesGraph');
-// TODO: (williamjallen) Replace this /ajax route with an equivalent /api route
-Route::get('/ajax/getsubprojectdependencies.php', 'SubProjectController@ajaxDependenciesGraph');
+Route::get('/viewSubProjectDependenciesGraph', 'SubProjectController@dependenciesGraph');
+Route::permanentRedirect('/viewSubProjectDependenciesGraph.php', '/viewSubProjectDependenciesGraph');
```

Comment:
If you're going to rename the route, I suggest something like this:
`/projects/<project ID>/subprojects/dependencies`